src/components/DisbursementInstructions/index.tsx

```diff
@@ -104,7 +104,51 @@ export const DisbursementInstructions: React.FC<DisbursementInstructionsProps> =
   return (
     <Card>
       <div className="DisbursementInstructions__titleWrapper">
-        <InfoTooltip infoText="Upload a list of recipients you wish to pay">
+        <InfoTooltip
+          infoText={
+            <div className="Note">
+              File format
+              <ul>
+                <li>
+                  <span className="Note__emphasis">
```

Comment:
Can we apply the `Note__emphasis` style directly to `li`? We could also create a data object with all the items and then map through them here (it will make sure we don't make mistakes in the future if we need to update the item style).

```typescript
{noteItems.map((n) => <li key={n.id}>{n.text}</li>)}
```